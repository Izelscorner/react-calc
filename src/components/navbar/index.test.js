import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './index';
import toJson from 'enzyme-to-json';

describe('<Logo />', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = shallow(<Navbar  />);
  });

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders the logo and src', () => {
    const navBar = wrapper.find('.nav-bar');
    expect(navBar).toHaveLength(1);
  });

});
