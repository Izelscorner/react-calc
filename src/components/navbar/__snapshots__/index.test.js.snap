// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> matches snapshot 1`] = `
<nav
  className="nav-bar"
/>
`;
